### DISCUSSION TITLE
Add support for module-path

### DISCUSSION THREAD

**Discussion by Thrameos** [jpype-project/jpype#1397] [Ideas]
Recently I have been attempting to use JavaFX from within JPype, but this requires the use of modules.   Unfortunately, we have never created the same interface for modules that we did for classpath and thus one has to specify everything manually.

**Comment by astrelsky** [comment]
This should be pretty simple to do. You can do it by providing the `--modulepath` with the vmargs or at runtime using the approach in the example. https://docs.oracle.com/javase/9/docs/api/java/lang/ModuleLayer.html

The only time I've ever used modules is when trying to exploit bluray disc java, so I don't know that much.

**Comment by Thrameos** [comment]
Fully agree.  Though I do think we should make it similar to classpath taking a list.

```
jpype.startJVM(modulepath=["path1","path2"])
```

Other wise we risk exposing them to the fact that Java uses different path separators for other systems.   Though this would be trivial to add I am going to work on it post 1.5.2 release.   I am trying to make use of the Javascript engine in Webview and there are a number of support routines that will be needed to make the suers life easier.   I am thinking that would be good items for a 1.6.0 release.
